src/Components/design/Footer/Footer.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Footer.css";
+
+export default function Footer(props) {
+  return (
+    <div className="footer">
+      <div className="footer-content">{props.footerText}</div>
+    </div>
+  );
+}
+
+Footer.propTypes = {
+  className: PropTypes.string,
```

Comment:
we don't seem to be using this prop, so maybe we remove the prop type or use it with clsx similar to other components?